dev/com.ibm.websphere.appserver.features/visibility/public/pages-4.0/io.openliberty.pages-4.0.feature

```diff
@@ -37,9 +37,9 @@ Subsystem-Name: Jakarta Server Pages 4.0
   com.ibm.websphere.appserver.eeCompatible-11.0, \
   io.openliberty.jakarta.pages-4.0, \
   io.openliberty.expressionLanguage-6.0
--bundles=com.ibm.ws.org.eclipse.jdt.core, \
+-bundles=io.openliberty.org.eclipse.jdt.core.java17; require-java:=17, \
```

Comment:
This needs to eventually be updated to the Java 21 version correct? If so can you open an issue and associate it with the Pages 4.0 Epic?

Reply:
No need. this java17 means it was built with java 17, but the code within has been updated to built JSPs with Java 21.

So you'd need to run on Java 21 in the first place for the JSPs to work (if setting javaSourceLevel to 21)